README.md

```diff
@@ -32,3 +32,7 @@ $extension->addToRunner($runner);
 ```
 
 ## Use it
+
+## License
+phpunit-extension is released under the BSD-3-Clause License. See the bundled LICENSE file for details.
```

Comment:
This is `prophecy-extension`, not `phpunit-extension` :wink:. What a troll 😛.

Also, can you put `prophecy-extension` and `LICENSE` inside a markdown code delimiters (\`)?